.github/workflows/publish.yml

```diff
@@ -20,11 +20,16 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Remove hardcoded MSRV
+        # Okay to build examples using the latest Rust.
+        # Fixes issues building wasm-bindgen-cli
+        run: rm -f rust-toolchain
```

Comment:
You can do `cargo +stable install wasm-bindgen-cli` to override it for a specific command

Reply:
Nice! I've used +nightly for things before but I hadn't connected the dots that I could use it to override the rust-toolchain file (that file is also new to me). Learned some things, thanks!